examples/gallery/ui/pages/text_edit_page.slint

```diff
@@ -10,6 +10,7 @@ export component TextEditPage inherits Page {
     description: @tr("Similar to LineEdit, but can be used to enter several lines of text. The widget can be imported from \"std-widgets.slint\".");
 
     HorizontalBox {
+        vertical-stretch: 1;
         GroupBox {
             vertical-stretch: 0;
```

Comment:
I believe the vertical-stretch here and in the other GroupBox are counter-productive, are they not?

Reply:
yeah the `vertical-stretch: 0;` is not needed. Maybe an old copy paste fragment.

Reply:
I have removed it from the groupbox